lib/core_dom/resource_url_resolver.dart

```diff
@@ -39,7 +39,15 @@ class ResourceUrlResolver {
   static final NodeTreeSanitizer _nullTreeSanitizer = new _NullTreeSanitizer();
   static final docForParsing = document.implementation.createHtmlDocument('');
 
-  static Node _parseHtmlString(String html) {
+  static String _getBaseUri() {
+    if (Uri.base.authority.isEmpty) {
+      throw "Uri.base.authority is Empty";
+    } else {
+      return "${Uri.base.scheme}://${Uri.base.authority}" + ('/');
```

Comment:
You can get rid of the manual string concatenation here and put the "/" inside the 1st string like this:

```dart
return "${Uri.base.scheme}://${Uri.base.authority}/";
```